doc/architecture/system-and-security-architecture.rst

```diff
@@ -417,20 +417,20 @@ For a detailed discussion of firewall configuration, see
 Production and Development Images
 =================================
 
-By default, building an image results in something that is locked-down
-and secure. This is how real products should be built. Unless some
-kind of application gets installed during image creation, one cannot
-do much with the running image (no user interface, no way to log into
-the system).
+By default, real product images should be built locked-down
```

Comment:
OK.  Thanks for the comment.  Sometimes editing to make something read better can, like this, subtly alter the meaning.  I'll try again...